homeassistant/components/lcn/__init__.py

```diff
@@ -102,7 +102,7 @@
     vol.Required(CONF_PORT): cv.port,
     vol.Required(CONF_USERNAME): cv.string,
     vol.Required(CONF_PASSWORD): cv.string,
-    vol.Optional(CONF_SK_NUM_TRIES, default=3): cv.positive_int,
+    vol.Optional(CONF_SK_NUM_TRIES, default=0): cv.positive_int,
```

Comment:
Should we mention this as a breaking change?

Reply:
I don‘t think it‘s worth it. Most users don‘t use these segment couplers.
And the documentation also mentions this.